src/app/screens/App/screens/Home/components/BookList/index.js

```diff
@@ -9,3 +10,7 @@ export default function BookList(props) {
     </div>
   );
 }
+
+BookList.propTypes = {
+  books: PropTypes.arrayOf(PropTypes.object).isRequired
```

Comment:
is better declare the shape of the object.
use Proptypes.shape({ .... }) instead of Proptypes.object